JensonChoi/moodify
# Art generation

**Changes:**
Added functionality for generating image, using OpenAI API, based off of mood passed in query/parameters value.

**Testing in Postman:**
1. follow instructions in `server/README.md` to get an OpenAI API key.
2. start server.
3. make GET Postman request to endpoint `/generate_art` with a `mood={insert what you want an image of}` key-value parameter in query params section (see image below).
<img width="904" alt="image" src="https://user-images.githubusercontent.com/41043008/221700390-aaa42d63-1579-4fe2-bc42-c32913c3b922.png">
4. send the request – it will take around 30 seconds to return. the request should return with a URL: 
<img width="851" alt="image" src="https://user-images.githubusercontent.com/41043008/221700559-8e8f02b0-72a4-4c1b-9038-9dafd2c3d4fe.png">
5. Open this URL in the browser to get your image.
<img width="1249" alt="image" src="https://user-images.githubusercontent.com/41043008/221700697-b083f1fa-3797-4d17-b764-a594a9e70d62.png">
